### PR TITLE
Initial components

### PR DESCRIPTION
**Pull request by DanteAlonsoHT**
# 📚   Bookstore: initialize the project with components 🧑‍🚀 

For this repository, I've created a project to create a React project using Redux, the main objective is to learn the way to create an application to store, use actions and reducers in React, to understand the concepts about state management, and connect React and Redux.

- [x] - Remove all unnecessary files
- [x] - Create components folder
- [x] - Create components/dataBooks.js
- [x] - Create components/FormBooks.js
- [x] - Create components/ListBooks.js
- [x] - Create components/Book.js
- [x] - Create components/pages/Home.js
- [x] - Create components/pages/Categories.js
- [x] - Create redux folder
- [x] - Create redux/books/books.js
- [x] - Create redux/categories/categories.js
- [x] - Create redux/configureStore.js
- [x] - Update App.js
- [x] - Update index.js
- [x] - Fix linters 
- [x] - Add descriptive README file

### CODE REVIEW

File: src/components/ListBooks.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+          books.map((book, index) => (
+            <div key={book.title} className="list-item" id={index}>
+              <div className="left">
+                <strong>{book.category}</strong>
+                <h2>{book.title}</h2>
+                <p>{book.author}</p>
+                <div className="interact">
+                  <span>Comments</span>
+                  |
+                  <span>Remove</span>
```
Comment (eliasecasta) [inline src/components/ListBooks.js]: - [x] Please make this a button to remove books to comply with the project requirements.

File: src/components/ListBooks.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+const ListBooks = (props) => {
```
Comment (eliasecasta) [inline src/components/ListBooks.js:3]: - [x] You are requested to have a book component and a booklist component, in the booklist you will map through all the books that you may receive and create a book component for each one, right now, you are doing both things in the same place, please divide this component in order to comply with the project requirements.

File: src/components/FormBooks.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      <option selected disabled>Action</option>
+      <option value="ScienceFiction">Science Fiction</option>
+      <option value="Drama">Drama</option>
+      <option value="Economy">Economy</option>
+      <option value="Romance">Romance</option>
+    </select>
+
+    <input type="submit" value="ADD BOOK" />
+
+  </form>
```
Comment (eliasecasta) [inline src/components/FormBooks.js]: - [x] This form should also accept an author input, please add it to comply with the project requirements.

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @DanteAlonsoHT, I am Elias, your reviewer for this PR

Well done so far with your project!

Your project complies with almost all the requirements, great job so far!

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [x] Your project is rendering a blank page when I open it, please take a second look at this to comply with the project requirements.
- [x] Please check the comments below for other things missing.

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by rukundoeric — APPROVED** [review]
Hi @DanteAlonsoHT,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**